crossfilter/universe
# Latest version

- Updated universe to latest crossfilter and reductio
- remove data with predicate as per https://github.com/crossfilter/crossfilter/wiki/API-Reference#wiki-crossfilter_remove
- make sure active filters are reapplied weh add/remove data
- add build version to the package
